src/main/java/com/passbook/sparkeighteen/peristence/POJO/PaginatedResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.passbook.sparkeighteen.peristence.POJO;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class PaginatedResponse<T> {
+
+    private List<?> content;
```

Comment:
What does List<?>

Reply:
actually, it is a generic type. it can accept any type of set assignment, but after assignment, you cant add elements to it, but you can remove and clear, not an immutable set.
List<?> is pronounced "List of unknown." it is convenient because it's flexible, but you cant shove random objects and pull them out of groups of unknowns with total impunity.

Reply:
@Bhavesh-SP18 Ghaatak english ka pradarshan :)